### DISCUSSION TITLE
custom optimizer

### DISCUSSION THREAD

**Discussion by uttamsuman** [lululxvi/deepxde#1623] [Q&A]
Is it possible to write a custom optimizer in deepxde. @lululxvi 
If yes how?

**Comment by lululxvi** [comment]
It is a question of TensorFlow or PyTorch.

**Comment by uttamsuman** [comment]
tensorflow

**Comment by lululxvi** [comment]
You can find an example here https://github.com/lululxvi/deepxde/pull/1661
